src/mon/MonCommands.h

```diff
@@ -1061,7 +1061,8 @@ COMMAND("osd pool create "
 	"name=autoscale_mode,type=CephChoices,strings=on|off|warn,req=false "
 	"name=bulk,type=CephBool,req=false "
 	"name=target_size_bytes,type=CephInt,range=0,req=false "
-	"name=target_size_ratio,type=CephFloat,range=0|1,req=false",\
+	"name=target_size_ratio,type=CephFloat,range=0|1,req=false "\
+	"name=yes_i_really_mean_it,type=CephBool,req=false",
```

Comment:
isn't there a problem with the order of the lines? note the '\' that makes the "create pool" part of the previous line

Reply:
No - actually there's an unneeded '\' in the previous line